src/main/java/de/tum/in/www1/artemis/repository/ExerciseGroupRepository.java

```diff
@@ -16,7 +16,8 @@
 @Repository
 public interface ExerciseGroupRepository extends JpaRepository<ExerciseGroup, Long> {
 
-    List<ExerciseGroup> findByExamId(Long examId);
+    @Query("SELECT e FROM ExerciseGroup e WHERE e.exam.id = :#{#examId}")
```

Comment:
You can remove this line with `@Query(...)` completely

Reply:
done